rye.go

```diff
@@ -24,6 +25,12 @@ type MWHandler struct {
 	beforeHandlers []Handler
 }
 
+// CustomStatter allows the client to log any additional statsD metrics Rye
+// computes around the request handler.
+type CustomStatter interface {
+	ReportStats(string, time.Duration, *http.Request, *Response) error
```

Comment:
Nit: I would name each arg in the interface - it makes the purpose more obvious.

As in:

`ReportStats(name string, elapsed time.Duration, req *http.Request, resp *Response) error`